### PR TITLE
Get Nairobi people email data

### PR DESCRIPTION
**Pull request by boswellgathu**
### What does this PR do?
It gets people emails data from slack

### CODE REVIEW

File: app/utils/get_emails.py [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    :param filename: Name of the file the data is to be written to
+    """
+    logging.info('Starting to write the selected data to emails.json')
+    with open(filename, 'w') as f:
+        # ensure_ascii makes sure that all ascii characters are left untouched
+        json.dump(persons_data, f, ensure_ascii=False, indent=4)
+    logging.info('File writen --> Done')
+
+
+def request_people_data():
```
Comment (RyanSept) [inline app/utils/get_emails.py:113]: I'd recommend you use the slack_client interface to make calls to the api that's been initialized in `app/__init__` over building the requests yourself.

Comment (boswellgathu) [reply app/utils/get_emails.py:113]: Fixed. I am now using the slack_client interface

File: app/utils/get_emails.py [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    """
+    Function to download data
+    :param params: dict of query strings for the url
+    :param url: url
+    :return: data - json data gotten
+    """
+    logging.info('Requesting for the data...')
+    if url:
+        res = slack_client.api_call(url, **params)
+        data = json.loads(res)
```
Comment (RyanSept) [inline app/utils/get_emails.py:47]: Is this necessary?
My understanding is that slack_client already gives you decoded json already.

File: app/utils/get_emails.py [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+    group_params = {'channel': channel_id}
+    try:
+        raw_group_data = get_data(group_url, group_params)
+        # no params since - {}
+        raw_channel_data = get_data(channel_url, {})
+        return raw_group_data, raw_channel_data
+    except requests.exceptions.MissingSchema as err:
+        logging.error('Error --> {}'.format(err))
+    except Exception as err:
+        if 'not_authed' in err.args:
```
Comment (RyanSept) [inline app/utils/get_emails.py:135]: I don't think this will be necessary now that we're using the slack_client interface.

**Comment by RyanSept** [comment]
This was great work. We're going to need to reconsider how to do ownership recognition differently then we'll see whether we'll need to reopen this MR

**Pull request closed without merging**